3.10/security-encryption.md

```diff
@@ -28,6 +28,11 @@ performance and resistance to side-channel attacks.
 
 The encryption feature is supported by all ArangoDB deployment modes.
 
+{% hint 'info' %}
+The ArangoGraph Insights Platform has encryption set on by default and cannot
```

Comment:
Maybe mention that this refers to encryption at rest (encrypted on disk) as well as in transit (encrypted communication)?

Reply:
yes, added it.